pkg/grpc/dapr/dapr_api.go

```diff
@@ -161,6 +163,19 @@ func (d *daprGrpcAPI) InvokeBinding(ctx context.Context, in *runtime.InvokeBindi
 	return r, nil
 }
 
+//todo  mosn isSecretAllowed  depend on  dapr！！
+func (d *daprGrpcAPI) isSecretAllowed(storeName string, key string) bool {
```

Comment:
why add this method?

Reply:
At present, the implementation relies on dapr, which includes permission configuration verification. At present, there is only configuration in mosn without permission control. An extension point is left here for later integration。

Reply:
So this is a function that of no use right now? Can we just delete it and write a TODO comment instead?
Or leave it blank like:
```
func (d *daprGrpcAPI) isSecretAllowed(storeName string, key string) bool {
       // TODO: add permission control
	return true
}

Reply:
Yes, leave a method definition，I'll go fix it

Reply:
> So this is a function of no use right now? Can we just delete it and write a TODO comment instead? Or leave it blank like:
> 
> ```
> func (d *daprGrpcAPI) isSecretAllowed(storeName string, key string) bool {
>        // TODO: add permission control
> 	return true
> }
> ```

modified！